apps/epic-web/src/inngest/functions/stripe/slack-daily-reporter.ts

```diff
@@ -232,17 +304,17 @@ export const slackDailyReporter = inngest.createFunction(
         }
       }
 
-      const labels = Object.keys(productData)
-      const data = Object.values(productData)
-      const formattedLabels = labels.map(
-        (name, index) => `${name}: ${formatCurrency(data[index])}`,
-      )
-
-      const total = data.reduce((sum, value) => sum + value, 0)
+      const entries = Object.entries(productData)
+      const filteredEntries = entries.filter(([_, value]) => value > 0)
+      const formattedLabels = filteredEntries.map(([name]) => name)
+      const data = filteredEntries.map(([_, value]) => value / 100)
+      const total = filteredEntries.reduce((sum, value) => sum + value[1], 0)
       const formattedTotal = formatCurrency(total)
 
+      const currentMonth = new Date().toLocaleString('default', {month: 'long'})
+
       const chartData = {
-        type: 'doughnut',
+        type: 'bar',
```

Comment:
I had to change the type of graph - a pie chart was not UX friendly when there are so many products and specially when Epic React has such an advantage over the other products, it wasn't legible. I used some min length in this type and it looks better